### PR TITLE
EBR-78 Windows build

### PR DESCRIPTION
**Pull request by AdrianValentinCiu**

### CODE REVIEW

File: tvb_build/docker/Dockerfile-build [hunk 1]
```diff
@@ -4,8 +4,7 @@ ARG TVB_USER=tvb_user
 ARG USER_HOME=/home/$TVB_USER
 ARG TVB_STORAGE=$USER_HOME/TVB_STORAGE
 ARG KEYCLOAK_CONFIG=$USER_HOME/keycloak_config
-ARG JUPYTER_CONFIG=$USER_HOME/.jupyter
-
+ARG JUPYTER_CONFIG=$USER_HOME/.jupyter>
```
Comment (liadomide) [inline tvb_build/docker/Dockerfile-build:7]: This this change ?

File: tvb_build/docker/Dockerfile-win [hunk 2]
```diff
@@ -27,6 +26,7 @@ RUN activate tvb-run && python setup.py develop
 
 COPY requirements_group requirements.txt
 RUN activate tvb-run && pip install -r requirements.txt
+RUN pip install -U scipy
```
Comment (liadomide) [inline tvb_build/docker/Dockerfile-win:29]: do we still need the pip install -U scipy if we removed the allensdk from req file ?

Comment (AdrianValentinCiu) [reply tvb_build/docker/Dockerfile-win:29]: I don't think so. I will remove it and build it again.

File: tvb_build/docker/Dockerfile-build (hunk 1 above)
Comment (AdrianValentinCiu) [reply tvb_build/docker/Dockerfile-build:7]: I will revert this change to the original one.

**Comment by liadomide** [comment]
#732 will redo this in another branch

**Pull request closed without merging**